tests/pyccel/test_pyccel.py

```diff
@@ -377,6 +388,14 @@ def test_expressions(language):
             [float]*3 + [int]*2 + [float]*2 + [int]*5 + [complex] + [bool]*9
     pyccel_test("scripts/expressions.py", language=language,
                 output_dtype = types)
+#------------------------------------------------------------------------------
+def test_highorder():
+    pyccel_test("scripts/runtest_highorder_functions.py",
+            dependencies = "scripts/highorder_functions.py",
+            compile_dep_with_pyccel = False,
+            output_dtype = [int,int,float,float,float,int,float,
+                float, int])
```

Comment:
This test should work in fortran and c

Reply:
we are already running the same tests in epyccel  for c, I think there is no need to run them twice .?

Reply:
So then do you think we should remove this test once the epyccel version can be activated for fortran? That sounds ok to me

Reply:
I agree

Reply:
I have added it to the list of things to do for issue #507